deploy/scripts/src/authentication/test.ts

```diff
@@ -443,18 +417,69 @@ export function initializeJourney(groups: readonly string[]): Response {
   })
 }
 
+export function orchStubSubmit(groups: readonly string[]): Response {
```

Comment:
Is this function replacing the `initializeJourney()` function?

Reply:
Yes

Reply:
Okay I suggest removing that function as part of this PR in that case

Reply:
Done